client/src/components/StrapGuide.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import Modal from "react-responsive-modal";
+
+class StrapGuide extends React.Component {
+  constructor (props) {
+    super(props);
+    this.state = {
+      open: false
+    }
+  }
+
+  onOpenModal () {
```

Comment:
The open and close Modal could probably be one function that toggles the state.

Reply:
Nice catch. I'll implement that suggestion.